components/Chat/Suggestions.tsx

```diff
@@ -3,10 +3,11 @@ import SuggestionButton from "../LandingPage/SuggestionButton";
 
 const Suggestions = () => {
   const { suggestions } = useChatProvider();
+  const limitedChatSuggestions = suggestions.slice(0, 2);
```

Comment:
Go further upstream.
- where are these being generated?
- can you modify the generation to only return `2 results` instead of `splicing` once we've already requested `4 results`?